src/CaseData.cpp

```diff
@@ -12,12 +16,41 @@
 
 using namespace amrex;
 
-CaseData::CaseData (const::std::string& fname)
+/*! \brief Constructor that initializes by reading in data from given file
+
+    See CaseData::InitFromFile()
+*/
+CaseData::CaseData (const::std::string& fname /*!< Filename to read case data from */)
 {
     InitFromFile(fname);
 }
 
-void CaseData::InitFromFile (const std::string& fname)
+/*! \brief Read case data from a given file
+
+    The case data file is an ASCII text file with three columns of numbers:
+    FIPS code, current number of cases, and cumulative number of cases till date.
+    (QDG: But data/CaseData/July4.cases has 2 columns!)
```

Comment:
@atmyers @tannguyen153 This function wants to read three columns from the file (FIPS code, current number of cases, and cumulative cases till date). But the example file (data/CaseData/July4.cases) has two columns.

Reply:
If file has two columns, j is junk, but num_cases2date is not used anyway.